samples/bluetooth/iso_combined_bis_and_cis/README.rst

```diff
@@ -0,0 +1,125 @@
+.. _bluetooth_isochronous_combined_bis_and_cis:
+
+Bluetooth: ISO Combined BIS and CIS
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® Combined BIS and CIS sample demonstrates the capability of
+a Controller to support both Broadcast Isochronous Stream (BIS) and Connected Isochronous Stream (CIS) simultaneously.
+It shows ISO-specific APIs to create a stream and to receive and send data over either CIS or BIS.
+The sample acts as a Central device that receives data over CIS and forwards the received data to a BIS.
+
+Note: The sample uses the Time of Arrival method to send data over ISO without using timestamps through a single stream (see :ref:`iso_providing_data`).
```

Comment:
Suggested change: `Note: The sample uses the Time of Arrival method to send data over ISO over a single stream (see :ref:`iso_providing_data`).`